package.json

```diff
@@ -32,6 +32,7 @@
     "ember-disable-prototype-extensions": "^1.0.0",
     "ember-export-application-global": "^1.0.2",
     "ember-test-container": "0.1.0",
+    "ember-getowner-polyfill": "1.0.0",
```

Comment:
This needs to be in `dependencies` section (not `devDependencies`).  Also, can you use `^1.0.0` instead of the exact version lock?